arch/arc/core/cache.c

```diff
@@ -39,6 +40,24 @@ size_t sys_cache_line_size;
 #define DC_CTRL_DIRECT_ACCESS        0x0  /* direct access mode  */
 #define DC_CTRL_INDIRECT_ACCESS      0x20 /* indirect access mode */
 #define DC_CTRL_OP_SUCCEEDED         0x4  /* d-cache operation succeeded */
+#define DC_CTRL_IM_MASK              BIT(6)         /* d-cache invalidate mode bit */
```

Comment:
I like the BIT(n) style, but now both styles are mixed in this file. Could you please either change the existing defines, or adopt to the earlier style for the new ones?